SU2_CFD/src/output/COutput.cpp

```diff
@@ -1099,7 +1099,10 @@ void COutput::SetScreen_Output(const CConfig *config) {
           PrintingToolbox::PrintScreenFixed(out, historyOutputPerSurface_Map.at(RequestedField)[0].value, fieldWidth);
           break;
         case ScreenOutputFormat::SCIENTIFIC:
-          PrintingToolbox::PrintScreenScientific(out, historyOutputPerSurface_Map.at(RequestedField)[0].value, fieldWidth);
+          // This line causes SCIENTIFIC output from Marker_Analyze to be written to screen twice after one another.
+          // Thus ruining the nice alignment of the columns in the screen output. Requires debugging.
+          // Try e.g. SURFACE_STATIC_TEMPERATURE with incompressible cases.
+          //PrintingToolbox::PrintScreenScientific(out, historyOutputPerSurface_Map.at(RequestedField)[0].value, fieldWidth);
```

Comment:
Concerning this review comment from @pcarruscag 

Maybe this is a way. Explaining the problem, giving example and commenting out the code. I am not sure to which degree the PerSurface output works (maybe only for history? i dont know) but as no regression test failed so at least for screen output this doesn't crash sth.